### DISCUSSION TITLE
Best strategy to clean nextjs build output

### DISCUSSION THREAD

**Discussion by LudoJH** [vercel/next.js#25382] [Help]
Hi there!

Until now, I was using these scripts to rimraf old build output before generating new one:
```
"scripts": {
    "clean": "npx rimraf .next"
    "dev": "next dev",
    "build": "yarn clean && next build",
    "start": "next start"
  },
```

However `yarn clean` also deletes `.next/cache/image` folder with all optimized image. Forcing next to re-optimized all images after every new build.

I wonder what is the best strategy, how do you manage this ? Do you delete .next folder excluding `.next/cache/image` folder ? Or is there any option to achieve this, I'm not aware of ? :D

**Comment by cdaringe** [comment]
i'm not clear on the .next/cache role at runtime either--i did add a `rm -rf` post-build. 🤞

**Reply by cdaringe** [reply]
related: https://github.com/vercel/next.js/discussions/23736
